Tasks/ServiceFabricDeploy/task.json

```diff
@@ -12,8 +12,8 @@
     ],
     "version": {
         "Major": 1,
-        "Minor": 2,
-        "Patch": 0
+        "Minor": 4,
+        "Patch": 7
```

Comment:
If you build the task - task.loc.json will have the similar change to update the version.